tests/python/unittest/test_contrib_operator.py

```diff
@@ -297,6 +297,18 @@ def f(x, a, b, c):
                                         [backward_expected],
                                         rtol=1e-2 if dtype is np.float16 else 1e-5,
                                         atol=1e-2 if dtype is np.float16 else 1e-5)
+def test_multibox_prior_op():
+    h = 561
+    w = 728
```

Comment:
Is it possible to not hardcode the test values?

Reply:
the input shape is designed specifically for the network